config.toml

```diff
@@ -1 +1,30 @@
-net_interface = "wls1" # Put here the network interface you want to use.
\ No newline at end of file
+net_interface = "wls1" # Put here the network interface you want to use.
+
+[[devices]]
+description = "Test Chromecast"
+mac_address = "AA:BB:CC:DD:EE:FF"
+pools = [1078, 3597]
+pools_description = [
```

Comment:
Idea: add a separate `[pools]` table where we put all the details about a pool (owner, description, ...). This way we don't repeat the same things about the pools for each device, and we also don't have to check that the `pools` and `pools_description` arrays are of the same size (we do have to check for the presence of a pool in the resulting map, however).
```toml
[[devices]]
description = "Test Chromecast"
mac_address = "AA:BB:CC:DD:EE:FF"
pools = [1078, 3597]

[pools]
    [pools.1078]
    description = "John Doe"

    [pools.1547]
    description = "Franck Dupond"

    [pools.2483]
    description = "Jean Poullain"

    [pools.3133]
    description = "Jean-Jacques Yves du Périgorac"

    [pools.3597]
    description = "Jane Doe"
```

```go
type brconfig struct {
	NetInterface string          `toml:"net_interface"`
	Devices      []bonjourDevice `toml:"devices"`
	Pools        map[string]pool `toml:"pools"`
}

type pool struct {
	Description string `toml:"description"`
}
```
Note: with this, the `Pools` map keys are strings, it would be better if they were integers

Reply:
@Gandem is the `description` field intended to be used somewhere in the code or is it only there to improve the readability of the config file?

Reply:
Only to improve readability